dspace-api/src/main/java/org/dspace/content/dao/impl/BitstreamDAOImpl.java

```diff
@@ -38,22 +42,27 @@ protected BitstreamDAOImpl() {
 
     @Override
     public List<Bitstream> findDeletedBitstreams(Context context) throws SQLException {
-        Criteria criteria = createCriteria(context, Bitstream.class);
-        criteria.add(Restrictions.eq("deleted", true));
-
-        return list(criteria);
+        CriteriaBuilder criteriaBuilder = getCriteriaBuilder(context);
+        CriteriaQuery criteriaQuery = getCriteriaQuery(criteriaBuilder, Bitstream.class);
+        Root<Bitstream> bitstreamRoot = criteriaQuery.from(Bitstream.class);
+        criteriaQuery.select(bitstreamRoot);
+        criteriaQuery.where(criteriaBuilder.equal(bitstreamRoot.get(Bitstream_.deleted), true));
+        return list(context, criteriaQuery, false, Bitstream.class, -1, -1);
 
     }
 
     @Override
     public List<Bitstream> findDuplicateInternalIdentifier(Context context, Bitstream bitstream) throws SQLException {
-        Criteria criteria = createCriteria(context, Bitstream.class);
-        criteria.add(Restrictions.and(
-            Restrictions.eq("internalId", bitstream.getInternalId()),
-            Restrictions.not(Restrictions.eq("id", bitstream.getID()))
-        ));
-
-        return list(criteria);
+        CriteriaBuilder criteriaBuilder = getCriteriaBuilder(context);
+        CriteriaQuery criteriaQuery = getCriteriaQuery(criteriaBuilder, Bitstream.class);
+        Root<Bitstream> bitstreamRoot = criteriaQuery.from(Bitstream.class);
+        criteriaQuery.select(bitstreamRoot);
+        criteriaQuery.where(criteriaBuilder.and(criteriaBuilder.equal(bitstreamRoot.get(Bitstream_.internalId),
+            bitstream.getInternalId()),
+            criteriaBuilder.equal(bitstreamRoot.get(Bitstream_.id), bitstream.getID())
```

Comment:
This line looks like a mistake.  Current code looks to be a `not equal`, and this code is an `equal()`